ui/src/pages/frontsite/components/Series/SeriesUrls.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'semantic-ui-react';
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import _truncate from 'lodash/truncate';
+
+const Url = ({ truncate, url }) => {
+  const description = url.description || url.value;
+  return (
+    <a href={url.value}>
```

Comment:
If they are used for in-app navigation, maybe we should consider using a `Link`

Reply:
I double-checked now and these URLs will be external and not in-app navigation. But perhaps an `ExternalLink` component could be useful that also could open the link in a new tab.